tap_bing_ads/__init__.py

```diff
@@ -20,13 +20,15 @@
 import backoff
```

Comment:
@namrata270998 I don't see README.doc changes in this PR

Reply:
That's right, there were no parameters related to config added, so we haven't added for the request timeout